src/controllers/product.js

```diff
@@ -235,11 +235,40 @@ const orderRequest = async (req, res) => {
   }
 };
 
+const seeRequesters = async (req, res) => {
+  try {
+    const product = await ProductModel.findById(req.params.productId);
+    if (!product) {
+      return res.status(404).json({
+        message: 'Product not found',
+      });
+    }
+
+    if (String(product.publisher) !== req.user.userId) {
+      return res.status(401).json({
+        message: 'You are not authorized to perform this action',
+      });
+    }
+
+    if (product.postType !== 'Donate') {
+      return res.status(400).json({
+        message: 'This is not a donation product: no requesters',
+      });
+    }
+
+    const requesters = product.orderRequests;
```

Comment:
i think we should use populate here to see the requesters info, id is not enough at this point imo.